src/array/cpu/spmm_blocking_libxsmm.h

```diff
@@ -264,7 +264,8 @@ inline libxsmm_meltwfunction_opreduce_vecs_idx SpMMCreateLibxsmmKernel(
                (sizeof(IdType) == 8) ? LIBXSMM_DATATYPE_I64 : LIBXSMM_DATATYPE_I32, opredop_flags);
   }
   if (kernel == nullptr) {
-    LOG(FATAL) << "Failed to generate libxsmm kernel for the SpMM operation!";
+    LOG(FATAL) << "Failed to generate libxsmm kernel for the SpMM operation."
+                  "You can use API dgl.use_libxsmm(false) to diable it!";
```

Comment:
```suggestion
                  " To disable libxsmm, use dgl.use_libxsmm(false).";
```
Be mindful of your tone. Try avoid second person in system messages. Be aware of spelling errors.

Reply:
Fixed, I will be careful about these next time.